src/StripeTests/Entities/StripeCustomerTest.cs

```diff
@@ -0,0 +1,46 @@
+namespace StripeTests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+
+    using Newtonsoft.Json;
+    using Stripe;
+    using Xunit;
+
+    public class StripeCustomerTest : BaseStripeTest
+    {
+        [Fact]
+        public void Deserialize()
+        {
+            string json = GetFixture("/v1/customers/cus_123");
+            var customer = Mapper<StripeCustomer>.MapFromJson(json);
+            Assert.NotNull(customer);
+            Assert.IsType<StripeCustomer>(customer);
+            Assert.NotNull(customer.Id);
+            Assert.Equal("customer", customer.Object);
+        }
+
+        [Fact]
+        public void DeserializeWithExpansions()
+        {
+            string[] expansions =
+            {
+              "default_source",
+            };
+
+            string json = GetFixture("/v1/customers/cus_123", expansions);
+            var customer = Mapper<StripeCustomer>.MapFromJson(json);
+            Assert.NotNull(customer);
+            Assert.IsType<StripeCustomer>(customer);
+            Assert.NotNull(customer.Id);
+            Assert.Equal("customer", customer.Object);
+
+            // TODO: Fix this in stripe-mock as Account is not a valid Customer source.
```

Comment:
Flagging this one as it's weird. I wanted to make tests pass but we should fix stripe-mock